bluemira/geometry/optimisation.py

```diff
@@ -123,8 +127,8 @@ class GeometryOptimisationProblem(OptimisationProblem):
     def __init__(
         self,
         geometry_parameterisation: GeometryParameterisation,
-        optimiser: Optimiser = None,
-        objective: OptimisationObjective = None,
+        optimiser: Optimiser,
+        objective: OptimisationObjective,
```

Comment:
As we discussed put this back for now, we need to have a full list of problems and rounded solutions before we fix things like this

Reply:
Done in [4139d03](https://github.com/Fusion-Power-Plant-Framework/bluemira/pull/1326/commits/4139d0318171f18fab176fc6ba940025965c84d1)